spec-main/api-ipc-renderer-spec.ts

```diff
@@ -192,4 +192,25 @@ describe('ipcRenderer module', () => {
       expect(result).to.deep.equal([])
     })
   })
-})
+
+  describe('after context is released', () => {
+    it('throws an exception', async () => {
+      const error = await w.webContents.executeJavaScript(`(${() => {
+        const child = window.open('', 'child', 'show=no,nodeIntegration=yes')! as any;
+        const childIpc = child.require('electron').ipcRenderer;
```

Comment:
@MarshallOfSound do you have any idea why `child.require` might not be defined here? This same logic works in 9.x+.

Reply:
@nornagon i suspect it's related to https://github.com/electron/electron/pull/19703